colibre/registration.py

```diff
@@ -225,7 +225,7 @@ def register_dust(self, catalogue):
     total_dust_fraction.name = f"$\\mathcal{{DTG}}${dust_frac_error}"
 
     # Compute total dust mass
-    total_dust_mass = total_dust_fraction * catalogue.masses.m_gas
+    total_dust_mass = total_dust_fraction * catalogue.masses.mass_gas
```

Comment:
Isn't it better to enable backward compatibility (i.e. to first try to fetch `mass_gas` and if that is not possible then switch to `m_gas`)?

Otherwise, we will need to rerun VR for a lot of runs if we want to update plots.

Reply:
We really need to use the updated VR. We can't keep using the old versions; too many important updates. 
We need to rerun it at least for all the cold gas properties. 

Is it then worth making the registration more complicated by adding backward compatibility?

Reply:
It is a new version of VR that gives `mass_gas` in place of m_gas, correct? (i.e. not the new config file or anything else)

For the backward compatibility question, I would for @JBorrow opinion. Personally, I am okay with either way.

Reply:
Well because this requires the updated version of VR to run anyway (for the HI, H2 masses), we can saftely remove the backwards compatibility here. I don't like the lack of deprecation time, but we can't really do anything else seeing as it was so broken before.